src/composables/multisigAccounts.ts

```diff
@@ -86,63 +87,62 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
     }
 
     const result = (await Promise.all(
-      rawMultisigData.map(async ({ contractId, gaAccountId, ...otherMultisig }) => {
-        try {
-          const contractInstance = await sdk.getContractInstance({
-            aci: SimpleGAMultiSigAci,
-            contractAddress: contractId,
-          });
-
-          const [
-            nonce,
-            signers,
-            version,
-            consensusResult,
-            balance,
-          ] = await Promise.all([
-            contractInstance.methods.get_nonce(),
-            contractInstance.methods.get_signers(),
-            contractInstance.methods.get_version(),
-            contractInstance.methods.get_consensus_info(),
-            gaAccountId ? sdk.balance(gaAccountId) : 0,
-          ]);
-
-          const consensus = consensusResult.decodedResult as any;
-
-          if (consensus.tx_hash) {
-            consensus.tx_hash = Buffer.from(consensus.tx_hash).toString('hex');
-          }
-          consensus.expiration_height = Number(consensus.expiration_height);
-          consensus.confirmations_required = Number(consensus.confirmations_required);
-          consensus.totalConfirmations = Number(consensus.confirmed_by.length);
-
-          const consensusLabel = (
-            `${consensus?.confirmed_by?.length}/${consensus.confirmations_required} ${i18n.t('common.of')} ${signers.decodedResult?.length}`
-          );
-
-          return {
-            ...camelcaseKeysDeep(consensus),
-            ...otherMultisig,
-            nonce: Number(nonce.decodedResult),
-            signers: signers.decodedResult,
-            version: version.decodedResult,
-            consensusLabel,
-            contractId,
-            balance: convertToken(balance, -MAGNITUDE),
-            address: gaAccountId,
-            multisigAccountId: gaAccountId,
-          };
-        } catch (error) {
+      rawMultisigData
+        .filter(({ version }) => version === SUPPORTED_MULTISIG_CONTRACT_VERSION)
+        .map(async ({ contractId, gaAccountId, ...otherMultisig }) => {
+          try {
+            const contractInstance = await sdk.getContractInstance({
+              aci: SimpleGAMultiSigAci,
+              contractAddress: contractId,
+            });
+
+            const [
+              nonce,
+              signers,
+              consensusResult,
+              balance,
+            ] = await Promise.all([
+              contractInstance.methods.get_nonce(),
+              contractInstance.methods.get_signers(),
+              contractInstance.methods.get_consensus_info(),
+              gaAccountId ? sdk.balance(gaAccountId) : 0,
+            ]);
```

Comment:
This is what I was talking about earlier. This information is not needed in the background. `signers, nonce, activeTx` data is needed only to display when the multisigAccount card is active. We could run the polls for these details only on the Active multisig tx review page( on demand)

Reply:
With your approach when you will swtich between multisig accounts you will not see an info instantly.

Reply:
It's a trade-off. You would otherwise simply call the dry run a hundred times and this data is not needed. 

Maybe we can fetch once in the beginning and later update on demand.

Reply:
I agree about `signers` and `nonce`, but `activeTx` is still needed on dashboard.

Reply:
Cool. May not be important for the MVP. PR looks good to me as of now.